### PR TITLE
patch(*): fix axle spacings

### LINKED ISSUE #49: axlespacings aren't listed as an array
Axle Spacings should be an array

### PR DESCRIPTION
**Pull request by gjulien-bjss**
## Ticket title

Fix axle spacings

Resolves https://github.com/dvsa/cvs-type-definitions/issues/49

## Changelog

- Put axle spacings in an array

### CODE REVIEW

**Review by naathanbrown — APPROVED** [review]

File: types/v3/tech-record/get/trl/complete/index.d.ts [hunk 1]
```diff
@@ -352,10 +352,15 @@ export interface TechRecordGETTRLComplete {
   techRecord_authIntoService_datePending?: string | null;
   techRecord_authIntoService_dateAuthorised?: string | null;
   techRecord_authIntoService_dateRejected?: string | null;
+  techRecord_dimensions_axleSpacing?: AxleSpacing[];
```
Comment (owen-corrigan) [inline types/v3/tech-record/get/trl/complete/index.d.ts:355]: if these arrays are optional do we need them to accept null also? or are they always an array or undefined?

Comment (gjulien-bjss) [reply types/v3/tech-record/get/trl/complete/index.d.ts:355]: yeah sure I can add the null "fields" to the get types, as we discussed earlier

Comment (gjulien-bjss) [reply types/v3/tech-record/get/trl/complete/index.d.ts:355]: Actually no, it can't be null from looking a the current backend validation.

Comment (gjulien-bjss) [reply types/v3/tech-record/get/trl/complete/index.d.ts:355]: image

Comment (gjulien-bjss) [reply types/v3/tech-record/get/trl/complete/index.d.ts:355]: but looking at that the `axles` field can't be null so I'll remove that

**Review by owen-corrigan — APPROVED** [review]

**Pull request merged**